### DISCUSSION TITLE
Run pluto with more threads?

### DISCUSSION THREAD

**Discussion by chunjiw** [fonsp/Pluto.jl#830] [Questions about using Pluto] [answered]
I noticed that in the source code, `Configurations.CompilerOptions` has a field `threads`; on my 12-core machine it defaults to 6. 

My question is, can I change this so that I can use more threads?

**Comment by chunjiw** [comment, marked as answer]
Turns out it is as simple as `Pluto.run(threads="11")` lol

**Comment by danvinci** [comment]
As a side note, might be trivial, but it'd be nice to have a consistent way of passing numerical parameters:

`Pluto.run(port=2345,threads=8)`

instead of:

`Pluto.run(port=2345,threads="8")`

Got a conversion error when using the former:
`ERROR: MethodError: Cannot `convert` an object of type Int64 to an object of type String`
